Source/Runtime/Measurement Service V1.lvproj

```diff
@@ -1,5 +1,7 @@
 ﻿<?xml version='1.0' encoding='UTF-8'?>
 <Project Type="Project" LVVersion="20008000">
+	<Property Name="NI.LV.All.SourceOnly" Type="Bool">true</Property>
```

Comment:
Given this project represents generated VIs, is there a reason to check the box here?

Reply:
The only reason I did it is to be consistent with the other projects (both the generated projects and non-generated projects). Also, if we ever add tests to these projects then they will be marked source-only be default, which is what we would want.